frontend/index.html

```diff
@@ -1,21 +1,27 @@
 <html>
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="third_party/vanillaSelectBox.css">
-    <link rel="stylesheet" href="https://unpkg.com/bootstrap-material-design@4.1.1/dist/css/bootstrap-material-design.min.css" integrity="sha384-wXznGJNEXNG1NFsbm0ugrLFMQPWswR3lds2VeinahP8N0zJw9VWSopbjv2x7WCvX" crossorigin="anonymous">
+    <link rel="stylesheet"
+        href="https://unpkg.com/bootstrap-material-design@4.1.1/dist/css/bootstrap-material-design.min.css"
+        integrity="sha384-wXznGJNEXNG1NFsbm0ugrLFMQPWswR3lds2VeinahP8N0zJw9VWSopbjv2x7WCvX" crossorigin="anonymous">
     <link rel="icon" href="Icons/Icons/google-spanner-logo.png" type="image/icon type">
-    <link rel='stylesheet' href='fonts/glyphicons-halflings-regular.woff2'></link>
+    <link rel='stylesheet' href='fonts/glyphicons-halflings-regular.woff2'>
+    </link>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap|Material+Icons" rel="stylesheet">
+    <link
```

Comment:
can you check the formatting here ? It looks like whitespaces have been introduced

Reply:
We are using Prettier here. It breaks the line if it has more than 80 (default) characters. this look good to me but we can update the config if you want to.